src/smart-components/order/order-item.js

```diff
@@ -27,91 +28,100 @@ import { getOrderIcon, getOrderPortfolioName } from '../../helpers/shared/orders
 import { createOrderedLabel, createUpdatedLabel, createDateString } from '../../helpers/shared/helpers';
 import createOrderRow from './create-order-row';
 
-const OrderItem = ({ item, isExpanded, handleDataItemToggle, portfolioItems }) => {
-  const { finishedSteps, steps } = createOrderRow(item);
-  const orderedAt = createOrderedLabel(new Date(item.ordered_at));
-  const updatedAt = createUpdatedLabel(item.orderItems);
+class OrderItem extends Component {
+  shouldComponentUpdate({ isExpanded }) {
```

Comment:
this could further be improved by returning true only when the item is being expanded, not when closed.

Reply:
I do not follow.

Reply:
If i understand your suggestion correctly it would mean the component would never close.